assets/src/blocks/SocialMedia/SocialMediaFrontend.js

```diff
@@ -0,0 +1,30 @@
+import { SocialMediaEmbed } from "./SocialMediaEmbed";
+
+export const SocialMediaFrontend = ({
+  title,
+  description,
+  embed_code,
+  social_media_url,
+  facebook_page_tab,
+  alignment_class,
+  className,
+  embed_type,
+}) => (
+	<section className={`block social-media-block ${className || ''}`}>
+		{title &&
```

Comment:
I'd prefer to use ternaries rather than && and you'll avoid to render `null` or `undefined` values. The logic is the same, It's just a prefrence.
For example these two cases:

`[].length && [].map...` if `[]` is `falsy`, then a `0` will be rendered on this component 
`[].length ? [].map... : null` if `[]` nothing will be rendered on this component. You're being explicit

Reply:
You can also do `!!title &&`, that's a lot easier on the eyes than ternary and no need to add `null` everywhere

Reply:
Updated! 🙂